presentation-compiler/test/dotty/tools/pc/tests/signaturehelp/SignatureHelpPatternSuite.scala

```diff
@@ -202,23 +202,6 @@ class SignatureHelpPatternSuite extends BaseSignatureHelpSuite:
          |""".stripMargin
     )
 
-  @Test def `pat4` =
```

Comment:
A. It is not working any more due to changes to range.
B. It was a bit useless as it would not be automatically opened and triggered, also the information is not that helpful, especially if you have more than single Apply like
`a :: b :: c :: Nil`